tests/test_config.py

```
@@ -112,40 +113,54 @@ def test_get_config_paths_linux(self):
             self.assertIn(os.path.expanduser("~/.mmrelay/config.yaml"), paths)
 
     @patch("mmrelay.config.os.makedirs")
-    @patch("mmrelay.config.platformdirs.user_config_dir")
-    def test_get_config_paths_windows(self, mock_user_config_dir, mock_makedirs):
+    @patch("mmrelay.paths.platformdirs.user_data_dir")
+    def test_get_config_paths_windows(self, mock_user_data_dir, mock_makedirs):
```

Ruff (0.14.14)

[warning] Unused method argument: `mock_makedirs`

(ARG002)